main.js

```diff
@@ -0,0 +1,46 @@
+/*
+Author: Anas Nasrallah.
+Purpose: Random User Page Generator.
+Date: 27.04.20
+*/
+
+/* This file is the mian file of the program. It handles the event 
+listeners and control the other files. */
+
+const APImanager = new APIManager;
+const renderer = new Renderer;
+localStorage.clear();
+localStorage.setItem('users', '[]');
+
+$('#load').on('click', function () {
+    APImanager.loadData();
+})
+
+$('#display').on('click', function () {
+    renderer.render(APImanager.data);
+})
+
+$('#save').on('click', function () {
+    users = JSON.parse(localStorage.users);
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"